src/utils/scaleDomainTransformers.ts

```diff
@@ -0,0 +1,38 @@
+///<reference path="../reference.ts" />
+
+module Plottable {
+
+  export module ScaleDomainTransformers {
+
+    /**
+     * Returns a translated domain of the input scale with a translation of the input translateAmount
+     * in range space
+     *
+     * @param {Scale.AbstractQuantitative<D>} scale The input scale whose domain is being translated
+     * @param {number} translateAmount The amount to translate
+     * @returns {D[]} The translated domain
+     */
+    export function translate<D>(scale: Scale.AbstractQuantitative<D>, translateAmount: number) {
+      var translateTransform = (rangeValue: number) => scale.invert(rangeValue + translateAmount);
+      return scale.range().map(translateTransform);
+    }
+
+    /**
+     * Returns a magnified domain of the input scale with a magnification of the input magnifyAmount
+     * in range space with the center point as the input centerValue, also in range space
+     *
+     * @param {Scale.AbstractQuantitative<D> scale The input scale whose domain is being magnified
+     * @param {number} magnifyAmount The amount to magnify
+     * @param {number} centerValue The center point of the magnification
+     * @param {boolean} centerInDomainSpace Specifying if the domain is in domain space (default = false)
```

Comment:
I think this parameter is confusing:
* Description seems to be incorrect
* Adds additional flexibility that probably won't be used (most `Interaction`s originate in pixel-space
* If the user is interesting in recovering the flexibility, they could just call `invert()` on their pixel value.